tests/cpp_unit_tests/test_atomic_mass_data.cpp

```diff
@@ -0,0 +1,31 @@
+#include <string>
+
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_floating_point.hpp>
+
+#include "openmc/atomic_mass_data.h"
+
+using namespace openmc::data;
+
+TEST_CASE("Test read and storing atomic data")
+{
+  AtomicData my_atomic_data();
+
+  std::string nucs[3] = {"Cl50", "Rh118", "Nd151"};
+  double expected_mass[3] = {50.008266, 117.930341116, 150.923839363};
+  double expected_binding[3] = {7651, 8322.8539, 8230.2741};
+  double expected_mass_excess[3] = {7700, -64886.840, -70943.183};
+
+  for (i < 0; i < 3; i++) {
```

Comment:
```suggestion
  for (int i < 0; i < 3; i++) {
```
Do we have to declare I before using it.

Reply:
yes ! this should be corrected in my last commits.